platform/efm32/inc/configurations.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2016 Silicon Laboratories, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __CONFIGURATIONS_H__
+#define __CONFIGURATIONS_H__
+
+/*******************************************************************************
+ * Universal configurations
+ ******************************************************************************/
+
+/* Memory boundaries */
+#define FLASH_ORIGIN 0x00000000
+
+/* Memory boundaries */
+#define SRAM_ORIGIN  0x20000000
+#define SRAM_OFFSET  0x0
```

Comment:
We allow vector relocation to SRAM, but we're relocating to a position _after_ the uvisor bss (`__start_vector_table__` in the .data section). This is because we need the DMA descriptors to end up before the vector table, since the ARM UDMA IP has strict alignment requirements.